wordpress-mobile/WordPress-Android
# [Prompts] Fix CTA text flickering in Prompts card

Fixes #18671 

It doesn't make sense to use a `FlexboxLayoutManager` here and it seems to be introducing issues with spacing, positioning, and measurement of the items when the card is updated multiple times. 

This causes the spacing between avatars to be inconsistent and was also causing the more visible issue of the text flickering between having 2 lines and 1 line.

I tried changing the Layout Manager to `LinearLayoutManager` fix this issue, but it seems the issue is actually related to the `TrainOfAvatarsAdapter` we are using and how it's measuring child items. I ended up creating a `View` wrapper around the `TrainOfIcons` Composable and using that paired with a `TextView` to achieve the same UI.

To test:
1. Open the Jetpack app with a blog site (containing a few published posts)
2. **Check** the Prompts card shows up on the Home dashboard
3. Tap "View all responses" to get to the reader tag screen
4. Go back to the dashboard
5. **Verify** the text and icons inside the Prompts card don't change

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
